extensions/assessmentExtension/schema.json

```diff
@@ -1,78 +1,51 @@
-          "type": "string",
-          "format": "uri"
-        },
-        "scoringMethodExampleDescription": {
-          "description": "The text of an example of the method or tool used to score the assessment.",
-          "type": "string"
-        },
-        "assessmentEvaluation": {
-          "description": "Link to studies or other information about research or calculations of reliability and validity for the assessment or the scoring methods.",
-          "type": "string",
-          "format": "uri"
-        },
-      }
+    "assessmentDescription": {
+      "description": " A description of the assessment.",
+      "type": "string"
     },
-    "AssessmentType": {
-      "description": "One of the following keywords: Exam, Performance or CreativeWork",
+    "assessmentType": {
+      "description": "The method used to conduct the assessment being referenced. Values are Exam, Performance or Artifact.",
       "type": "string",
-      "pattern": "^(Exam|Performance|CreativeWork)$"
+      "pattern": "^(Exam|Performance|Artifact)$"
     },
-  },
-  "properties": {
-    "overallDescription": {
-      "description": "A description of how the assessment activity is organized, particularly describing the relationship between multiple assessments included in the extension. For example, this could include information about an overall project that has multiple assessment components.",
+    "assessmentOutput": {
+      "description": "This field provides additional details about assessmentType. Values for assessmentOutput are expected to be words or phrases that describe the key features of the evidence that are produced in earning the badge.",
+      "type": "string"
+    },
+    "hasGroupParticipation": {
+      "description": "Completing the assessment activity being referenced requires two or more participants.",
+      "type": "boolean"
+    },
+    "hasGroupEvaluation": {
+      "description": "Participants in the assessment activity being referenced are scored as a group. ",
+      "type": "boolean"
+    },
+    "evaluationMethod": {
+      "description": "Information about how the assessment is scored.  What do the scores represent in a range of scores? If a rubric was used, what are the score ranges for each criteria?",
       "type": "string"
     },
-    "assessment": {
-      "type": "object",
-      "$ref": "#/definitions/AssessmentArray"
+    "assessmentExample": {
+      "description": "An example based on the assessment type.",
+      "type": "string",
+      "format": "uri"
     },
-  },
+    "scoringMethodExampleDescription": {
+      "description": "The text of an example of the method or tool used to score the assessment.",
+      "type": "string"
+    },
+    "assessmentEvaluation": {
+      "description": "Link to studies or other information about research or calculations of reliability and validity for the assessment or the scoring methods.",
+      "type": "string",
+      "format": "uri"
+    }
+  }
```

Comment:
(There's also a missing opening `{` at the top of the file. Somehow my comment on that didn't get into the review, probably forgot to save it.)